### ISSUE TITLE
What's the state of this?

### ISSUE THREAD

**Issue by josephwoodward** [kanekotic/Specflow.Rider#4]
I see this hasn't been touched in a while, what's the state of it? I was considering writing a plugin around this.

**Comment by alvarolorentedev** [comment]
There was a change of priorities on the work I was doing so the priority for this went down.
I have not focused on it for a few months, but I will be happy to refocus on this and get out the first release. 
will you be interested in both collaborating to finish a release @JosephWoodward ?

**Comment by josephwoodward** [comment]
Yes, certainly interested. Though I've never written a plugin for Rider before so I don't know how useful I'd be.

How do you test this in Rider after cloning the repo?

**Comment by 0xRapid** [comment]
I would love to help as-well with what I can. 
Lets get it done.

**Comment by alvarolorentedev** [comment]
Ok. Let me get my ideas straight on what is left and a few instructions for setting up the environment to test this and I will get back to both of you in this issue. Probably I will have time to do this by midweek.

**Comment by josephwoodward** [comment]
@kanekotic That would be amazing. Tried getting started but the docs around Rider plugins seems to be a bit of a mess at the moment.

**Comment by alvarolorentedev** [comment]
I will share later the docs I followed. In general what I am trying is to transpile the `.feature` into the `.cs` files. I think I will completely pivot out of that. and follow a simpler newer approach like in this post (that if you think can fulfill your needs you should follow).

**Comment by 0xRapid** [comment]
I'm not sure if you intended to post a link that directs us somewhere, anyhow I came across (https://kenbonny.net/2018/05/28/generating-specflow-files-in-rider/).
It transpile `.feature` files into it's `.feature.cs` by using SpecFlow.exe file after you are saving a feature file.
(If that's what you mean).

Or you are talking about the process of "Generate Step Definitions"? 
(https://kenbonny.net/2018/07/23/specflow-steps-generation-and-general-rider-changes/)

One of the thing that is pretty annoying at the moment is that you can't jump to step declaration from the feature file.

**Comment by josephwoodward** [comment]
Leaning on the SpecFlow.exe that's bundled in with the NuGet package seems the best way to go.

The real need for this plugin is to provide the necessary UI extensions within Rider to enable you to easily navigate between the step declaration files to the feature file, as well as a means of generating the .feature.cs files within Rider.

**Comment by alvarolorentedev** [comment]
yes, I agree with both of you now that the existence of the NuGet package is here. So I will reset this and lets focus as you both say in:
- Allow compiling in one click a `.feature` using the NuGet package.
- And improve the navigation.

**Comment by 0xRapid** [comment]
I think we are both still waiting for the configuration and the instructions by you to get it started :)

**Comment by TheDevelolper** [comment]
I'd like to help out but I can't seem the compile this.

**Comment by 0xRapid** [comment]
We are waiting aswell. :p
